src/main/resources/changesets/changelog-consent-98.0.xml

```diff
@@ -0,0 +1,12 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="98.0" author="grushton">
+        <addColumn tableName="dataset">
+            <column name="sharing_plan_document_name" type="varchar(255)"/>
+        </addColumn>
+        <rollback>
+            <dropColumn tableName="dataset" columnName="sharing_plan_document_name"/>
+        </rollback>
```

Comment:
just curious (does not affect approval): is this added so we can turn back changes in changesets? should i be adding rollback sections to all of my changesets moving forward?

Reply:
This is the failure case. If anything goes wrong, this is the solution to the error. Yes, we should usually have rollbacks in the cases where things can be rolled back.